### ISSUE TITLE
[FE] [Assistive tech and device support] UI components initiate a change of context when they receive user input. (11.13.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#31440]
#### VFS team name
Authenticated Experience

#### VFS product name
Notification Preferences

#### Point of Contact/Reviewers
Josh Kim (Accessibility)

---

### Platform Issue
UI components initiate a change of context when they receive user input.

### Issue Details
When an edit field is open and user attemps to remove a contact type from their page, the edit field is closed without their knowledge. Screen reader users may be confused as to why the field they were working in earlier was suddenly closed without their knowledge.

### Link, screenshot or steps to recreate
1. Log in to staging using test user 36 and turn on your screen reader
2. On personal and contact information, click "edit" for any contact type 
3. Then, click "remove" for another contact type
4. Cancel the modal, and confirm that the previous edit field is now closed without informing the user

### VA.gov Experience Standard
[Category Number 11, Issue Number 13](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.2_A

---

### Platform Recommendation
Either give users a heads up that removing a contact type will close any open edit states or leave the edit state open when a user attemps to remove a contact type.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by MarciMcGuireGCIO** [comment]
Adding note based on our discussion in grooming - we think a good approach here is to have the Remove button exhibit the same behavior as the Edit button when it's clicked while someone is editing another field.    Rather than presenting them with a modal asking them to acknowledge removal, it should display a modal advising they are already mid-edit elsewhere: 

![image](https://user-images.githubusercontent.com/73354907/150389393-ce00e313-712d-4632-9dd8-3f2ff76a8be5.png)

**Comment by MarciMcGuireGCIO** [comment]
@adamwhitlock1  - I see a PR and merge for this; is it OK to close?  Thx!

**Comment by adamwhitlock1** [comment]
Yep, should be good to close, it will be up on staging shortly.
